prioritize.py

```diff
@@ -0,0 +1,118 @@
+import os
+
+import click
```

Comment:
group the imports as follows:
1. python sys packges
2. python pip packages (including kili)
3. repo modules